components/list/list-item-mixin.js

```diff
@@ -146,7 +146,6 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			:host([_focusing-primary-action]) [slot="control-container"]::after,
 			:host([selected]:not([selection-disabled]):not([skeleton])) [slot="control-container"]::before,
 			:host([selected]:not([selection-disabled]):not([skeleton])) [slot="control-container"]::after,
-			:host([_show-add-button]) [slot="control-container"]::after,
```

Comment:
This keeps the border color when there is an add button rather than setting it to transparent. This is due to the fact we do still want the full border for nested lists.

I'm going to see if I can target this more specifically to at least just those cases. The impact of this is that you see a tiny bit of border behind the + icon in the add button.